source/import-pipeline-from-text.txt

```diff
@@ -12,30 +12,58 @@ Import Pipeline from Text
    :depth: 1
    :class: singlecol
 
-*New in version 1.15.0*
-
 You can import aggregation pipelines from plain text into
 the :ref:`Aggregation Pipeline Builder <compass-agg-builder>` to
 easily modify and execute your pipelines. Importing a plain text
 aggregation pipeline shows how each stage of the pipeline
 affects the output, and illustrates the effects of modifying specific
 pipeline stages using the Pipeline Builder's :guilabel:`Output` panes.
 
+Starting in Compass 1.35, the import pipeline button has been removed
+from the Compass GUI. The full functionality of importing pipelines 
+from text is still supported as part of the View Pipeline as 
+Text edit mode.
+
+
 Syntax
 ------
 
-The imported pipeline must be in the
-:manual:`MongoDB query language </tutorial/query-documents/>`
-(i.e. the same syntax used as the ``pipeline`` parameter of the
-:manual:`db.collection.aggregate()
-</reference/method/db.collection.aggregate/>` method). The imported
-pipeline must be an array, even if there is only one stage in the
-pipeline.
+The imported pipeline must be:
+
+- In the :manual:`MongoDB query language </tutorial/query-documents/>`,
+  this is the same syntax used as the ``pipeline`` parameter of the
+  :manual:`db.collection.aggregate()
+  </reference/method/db.collection.aggregate/>` method.
+- An array, even if there is only one stage in the pipeline.
 
 Procedure
 ---------
 
-.. include:: /includes/steps/import-pipeline-from-text.rst
+.. procedure::
+   :style: normal
+
+   .. step:: In the aggregation pipeline pane, click the 
+      :guilabel:`</> TEXT` toggle switch to enable text mode for 
```

Comment:
I left a similar comment on PR #446 but missed it here - can we use initial case `Text` here rather than all-caps? Same comment applies to other all-caps UI elements in this update.